examples/ex010_advection.jl

```diff
@@ -0,0 +1,54 @@
+# ---------------------------------------------------------------
+# advection example following Melvin, Staniforth and Thuburn
+# Q.J:R. Meteorol. Soc. 138: 1934-1947, Oct (2012)
+# non polynomial advection example using transplanted
+# quadrature formulas from conformal maps following Hale and 
+# Trefethen (2008) SIAM J. NUMER. ANAL. Vol. 46, No. 2
+# ---------------------------------------------------------------
+using LFAToolkit
+using LinearAlgebra
+using Polynomials
+
+# setup
+mesh = Mesh1D(1.0)
+P = 11;
+Q = P;
+collocate = false
+#mapping = sausage(29)
+#mapping = kosloff_tal_ezer(0.98)
+mapping = hale_trefethen_strip(1.4)
+basis = TensorH1LagrangeBasis(P, Q, 1, 1, collocatedquadrature = collocate, mapping = mapping)
+
+# frequency set up
+numbersteps = 250
```

Comment:
The standard amount should be enough.
```suggestion
numbersteps = 100
```